packages/clerk-js/src/ui/components/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -45,7 +45,10 @@ const EyeCircle = ({ width, height, ...props }: EyeCircleProps) => {
       <Icon
         elementDescriptor={descriptors.impersonationFabIcon}
         icon={Eye}
-        size={undefined}
+        sx={t => ({
```

Comment:
❓ Is this change intentional? Are we fixing a bug?

Reply:
This was intentional. I had to update the eye svg icon we had. The previous one had a predefined width, height and white color inside the svg file and it was used only in the Impersonation FAB.

Now we need the eye and eye-slash icons to match so it doesn't look weird when we mount one and unmount the other.  Then changes made in the FAB are there to ensure the new Icon has similar color and size to the icon used before.

Reply:
### Before 
<img width="280" alt="image" src="https://user-images.githubusercontent.com/19269911/234703180-2bca6823-aa10-40bf-a21e-6b908362e59d.png">

### After
<img width="286" alt="image" src="https://user-images.githubusercontent.com/19269911/234703263-c6ca7774-ff4a-4ed6-b9aa-fe297abdfde5.png">

As you seen they look similar even after the changes

Reply:
What about the `EyeSlash` icon?

Reply:
@desiprisg EyeSlash is already as it should